src/repl.ml

```diff
@@ -169,6 +183,13 @@ module Command = struct
     in
     Extension_commands.register ~id:Extension_consts.Commands.open_repl handler
 
+  let evalTextDecorationType : TextEditorDecorationType.t =
+    let bgcolor = get_repl_evalTextBackgroundColor () in
+    let render_options =
+      Vscode.DecorationRenderOptions.create ?backgroundColor:bgcolor ()
+    in
+    Vscode.Window.createTextEditorDecorationType ~options:render_options
```

Comment:
This should be a function so users can change the selection background color without having to restart the extension. Since it's a value currently, that means it's only updated once when the extension is started.

Reply:
I redefined it out as the `getEvalTextDecorationType` function.